### PR TITLE
Create docker image for cyclops

### PR DESCRIPTION
**Pull request by WheresWardy**

### CODE REVIEW

**Comment by iserko** [comment]
you should add a CMD too

**Comment by WheresWardy** [comment]
I did think that, but wasn't sure what it would be; I thought the command might be passed in (like with the web server containers) so was going to do it that way, or can it just be controlled entirely by the Dockerfile? I wasn't having much luck just passing the commad parameters for cyclops into it!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:trusty
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get -y install git curl libcurl4-openssl-dev build-essential && \
+    apt-get -y install python-dev python-pip python-virtualenv
+
+WORKDIR /opt
```
Comment (iserko) [inline Dockerfile]: starting here I think you should do:

```
ADD . /opt/cyclops
ENV PYTHONPATH=/opt/cyclops/
RUN pip install -r requirements.txt

EXPOSE 9000
WORKDIR /opt/cyclops
CMD python -m cyclops.server -vv -c cyclops.conf
```

you will need to add `requirements.txt` with the packages from `setup.py`

**Comment by iserko** [comment]
:+1:

**Pull request merged**